specification/hugr.md

```diff
@@ -1170,44 +1159,71 @@ The new hugr is then derived as follows:
 
 ###### `Replace`
 
-This takes as input:
-
-  - a set S of IDs of nodes in Γ, all of which are separated;
+This is the general subgraph-replacement method.
 
-  - a partial hugr G;
+A _partial hugr_ is a graph formed by a subset of nodes of a valid hugr together
+with a subset of their adjoining edges. It must not include a `Module` node.
 
-  - a map T from top(G) to IDs of container nodes in Γ\\S\*;
+Informally, a partial hugr is is a graph $G$ having the same node and edge types
+as a hugr but that may be missing some edges or nodes.
```

Comment:
nit: `or' -> `and/or`

Reply:
In fact I'm not sure this sentence adds anything to the previous. Although maybe reiterating that helps for it to sink in ;)